### DISCUSSION TITLE
zeabur上部署的MongoDB

### DISCUSSION THREAD

**Discussion by antameng** [xkdvSrPD/blog-next-view#39] [General]
zeabur 免费版上部署的MongoDB    是持久的嘛   为啥我过了一两天就连不上了 重启服务也重启不了

**Comment by xkdvSrPD** [comment]
好像不是的，免费用户只能部署serverless应用，你可以使用mongo atlas部署免费的mongo数据库

**Comment by antameng** [comment]
感谢回复，  在阿里云用docker启了一个MongoDB容器，暂时先用着
